src/client/utils.ts

```diff
@@ -26,11 +31,52 @@ export function getVConfig(): WorkspaceConfiguration {
 	return workspace.getConfiguration("v", uri);
 }
 
-export function getCwd() {
-	const folder = workspace.workspaceFolders[0];
+export function getCwd(uri?: Uri): string {
+	const folder = getWorkspaceFolder(uri || null);
 	return folder.uri.fsPath;
 }
 
+export function getWorkspaceFolder(uri?: Uri): WorkspaceFolder {
+	if (uri) return workspace.getWorkspaceFolder(uri);
+
+	const currentDoc = getCurrentDocument();
+	return currentDoc
+		? workspace.getWorkspaceFolder(currentDoc.uri)
+		: workspace.workspaceFolders[0];
+}
+
 export function getCurrentDocument(): TextDocument {
 	return window.activeTextEditor ? window.activeTextEditor.document : null;
 }
+
+export function arrayInclude(arr: Array<string>, search: string): number {
+	let ind = -1;
+	arr.forEach((sp, i) => {
+		if (sp.indexOf(search) != -1) {
+			ind = i;
+			return;
+		}
+	});
+	return ind;
+}
```

Comment:
This is already achievable using [`Array.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes):

~~~js
const list = ["Foo", "Bar", "Baz"];
list.includes("Bar") === true;
list.includes("Qux") === false;
~~~

Reply:
yeah.. just wanna get  index

Reply:
Then use [`Array.prototype.indexOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) or [`Array.prototype.findIndex`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex).

Reply:
`IndexOf()` return the index if exact match

Reply:
let index = ["Foo", "Bar", "Baz"].findIndex(str => str.includes("Bar"));
index === 1;